aws-athena/views/reporting-vw_assessment_roll_muni.sql

```diff
@@ -77,42 +77,92 @@ pin_counts AS (
         vptc.year,
         vptc.major_class,
         stages.stage
+),
+
+/* This CTE solely provides lagged values by year and stage by PIN. Because
+parcel universes change year-to-year (PINs are activated/deactivated, change
+class), building "prior phase" stats by aggregating lagged values will lead to
+small disgareements between a given year's "prior phase" stats for its mailed
```

Comment:
**issue (blocking):** I would clarify this comment a little bit. I found it unclear until I read the actual SQL below to see delta calculation.